### ISSUE TITLE
Delete this repository

### ISSUE THREAD

**Issue by mkllnk** [coopdevs/ofn-de-secrets#2]
@enricostano @sauloperez This repository has been replaced by https://github.com/openfoodfoundation/ofn-secrets and is now outdated. I propose to delete this repository to avoid confusion and remove the vault from the public. Do you agree?

The same probably applies to katuma-secrets. The us credentials have not been copied yet.

**Comment by sauloperez** [comment]
Yep, I need to work on that for Katuma. I believe I moved them all to the new repo.

**Comment by enricostano** [comment]
You can just archive it maybe.
